src/query/ast/tests/it/testdata/stmt.txt

```diff
@@ -2937,9 +2937,9 @@ UseDatabase {
 
 
 ---------- Input ----------
-create catalog ctl type=hive connection=(url='<hive-meta-store>' thrift_protocol='binary');
+create catalog ctl type=hive connection=(url='<hive-meta-store>' thrift_protocol='binary' warehouse='default');
 ---------- Output ---------
-CREATE CATALOG ctl TYPE=HIVE CONNECTION = ( thrift_protocol = 'binary', url = '<hive-meta-store>' )
+CREATE CATALOG ctl TYPE=HIVE CONNECTION = ( thrift_protocol = 'binary', url = '<hive-meta-store>', warehouse = 'default' )
```

Comment:
why does catalog requires to bound with a warehouse?

Reply:
Just a test. In old test https://github.com/databendlabs/databend/blob/f40b8c303b558a87956785ce128f8aed8d752a07/tests/suites/3_stateful_iceberg/00_rest/00_0000_create_and_show.sh#L15 it already exists.

Reply:
> why does catalog requires to bound with a warehouse?

Hi @flaneur2020, catalog does need to bound with a warehouse.

Reply:
noticed this is not the "warehouse" we described in the sense of databend cloud, but more like a namespace identifier inside a catalog 😲